release-notes/current.yaml

```diff
@@ -17,6 +17,7 @@ new features: |
   Added support for trusted CIDRs in the ClientIPDetectionSettings API
   Added support for sending attributes to external processor in EnvoyExtensionPolicy API
   Added support for patching EnvoyProxy.spec.provider.kubernetes.envoyHpa and EnvoyProxy.spec.provider.kubernetes.envoyPDB
+  Added Lua support in EnvoyExtensionPolicy and HTTPRouteFilter APIs
```

Comment:
Nit: We don't need this since the API is currently hiden. let's add release note after implementing the Lua filter.

Reply:
Done!